### ISSUE TITLE
update to 1.0

### ISSUE THREAD

**Issue by WenmuZhou** [meijieru/crnn.pytorch#182]
Will support pytorch 1.0?

**Comment by shuangyueliao** [comment]
no

**Comment by leonzhu211** [comment]
ivery few modification for python3 + pytorch 1.0 environment.
ctc loss already update for pytorch1.0
https://github.com/t-vi/warp-ctc.git
it should be compled with gcc 4.9.0+

then, 
from warpctc import CTCFunction as CTCLoss

**Comment by WenmuZhou** [comment]
when I use ctcloss in pyotrch 1.0, I get nan after some epochs

**Comment by leonzhu211** [comment]
@WenmuZhou learning rate is too large

**Comment by TyrionChou** [comment]
> when I use ctcloss in pyotrch 1.0, I get nan after some epochs

是否是学习率太高导致的呢？

**Comment by WenmuZhou** [comment]
in torch 1.4,its ok
